Docs/Book/Install.md

```diff
@@ -2,6 +2,61 @@
 
 Below a number of installation recipies is presented, with varying degree of complexity.
 
+### Cross-platform under anaconda python (fastest install)
+
+### Introduction to anaconda
+
+Conda is an open-source, cross-platform, software package manager. It supports the packaging and distribution of software components, and manages their installation inside isolated execution environments. It has several analogies with pip and virtualenv, but it is designed to be more "python-agnostic" and more suitable for the distribution of binary packages and their dependencies.
+
+### How to get conda
+
+The easiest way to get Conda is having it installed as part of the [Anaconda Python distribution](http://docs.continuum.io/anaconda/install.html). A possible (but a bit more complex to use) alternative is provided with the smaller and more self-contained [Miniconda](http://conda.pydata.org/miniconda.html). The conda source code repository is available on [github](https://github.com/conda) and additional documentation is provided by the project [website](http://conda.pydata.org/). 
+
+### How to install RDKit with Conda
+
+Creating a new conda environment with the RDKit installed using these  packages requires one single command similar to the following::
+
+  $ conda create -c <channel-url> -n my-rdkit-env rdkit
+
+where '<channel-url>' is to be replaced with the URL of the package repository where the packages have been placed for distribution.
```

Comment:
what about replacing the <channel-url> placeholder with the actual rdkit channel on anaconda.org (or providing an example using this channel as a reference)?